src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+
+import { BsModalRef } from 'ngx-bootstrap';
+
+import { SummaryService } from '../../../shared/services/summary.service';
+
+@Component({
+  selector: 'cd-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.scss']
+})
+export class AboutComponent implements OnInit {
+  version: string;
+
+  constructor(public modalRef: BsModalRef, private summaryService: SummaryService) {}
+
+  ngOnInit() {
+    this.summaryService.get().subscribe((summary: any) => {
```

Comment:
We could remove a new request to the backend if, instead of calling `get`, we subscribe to the service.
We might need to replace `Subject `observable with `BehaviorSubject`, so we can obtain the value as soon as we  subscribe.